fe/src/pages/campaigns/[id].vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <div
+    class="flex flex-col gap-6 py-6 px-36 md:px-44 min-h-screen lg:px-48 bg-linear-gradient-white-to-light overflow-auto"
+  >
+    <div class="flex gap-[22px]">
+      <div class="w-full h-[420px]">
+        <img
+          :src="imgSrc"
+          alt="Boy writing on paper"
+          class="object-cover w-full h-[420px]"
+        />
+      </div>
+
+      <div class="flex flex-col gap-[24px]">
+        <DetailCard label="Days left" :content="daysLeft" />
+        <DetailCard label="Donations" :content="`${donations} DCAPP`" />
+        <DetailCard label="Total backers" :content="totalBackers" />
+      </div>
+    </div>
+
+    <div class="bg-white p-4 flex gap-6 sm:flex-col lg:flex-row">
+      <div class="flex flex-col gap-6 sm:w-full lg:w-[654px] xl:w-full">
```

Comment:
Why sm and xl is w-full while lg is not? Weird that the middle screen has different width?

Reply:
changed `lg:w-[654px] xl:w-full` to `lg-full` only